handlers/update.go

```diff
@@ -128,6 +129,17 @@ func updateSpec(request *requests.CreateFunctionRequest, spec *swarm.ServiceSpec
 	}
 
 	spec.TaskTemplate.ContainerSpec.Secrets = secrets
+	spec.TaskTemplate.ContainerSpec.ReadOnly = request.ReadOnlyRootFilesystem
+
+	spec.TaskTemplate.ContainerSpec.Mounts = removeMounts(spec.TaskTemplate.ContainerSpec.Mounts, "/tmp")
```

Comment:
This was needed in faas-netes, but is it also needed in Swarm?

Reply:
This is simply updating the match the request, if it was previously a `ReadOnlyRootFilesystem = true` and now it is `ReadOnlyRootFilesystem = false` the spec needs to update appropriately.  The use case is exactly the same as in faas-netes, hence why it looks similar